src/App.tsx

```diff
@@ -1,9 +1,20 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const getFilterText = (text: string, searchQuery: string) => {
+    return text.toLowerCase().includes(searchQuery.trim().toLowerCase());
+  };
```

Comment:
Let's move this function out of the component. We don't need to create on each re-render